core/types/celo_transaction_signing_forks.go

```diff
@@ -0,0 +1,106 @@
+package types
+
+import "github.com/ethereum/go-ethereum/params"
+
+var (
+	// celoForks is the list of celo forks that are supported by the
+	// celoSigner. This list is ordered with more recent forks appearing
+	// earlier. It is assumed that if a more recent fork is active then all
+	// previous forks are also active.
+	celoForks = forks{&cel2{}, &celoLegacy{}}
```

Comment:
Can you add some test to make sure these are ordered, because it's assumed in line 19.

Reply:
Ok added a ticket for testing and explicitly mentioned tests for correct fork ordering - https://github.com/celo-org/op-geth/issues/168

Reply:
As we discussed, running [this](https://github.com/celo-org/celo-blockchain-planning/issues/363) test across all the alfajores history should give us fairly high confidence that the signing is in order and the work for that is almost done.